packages/core/src/did/Did.spec.ts

```diff
@@ -47,6 +47,16 @@ describe('DID', () => {
   })
 
   it('query by address w/o documentStore', async () => {
+    blockchainApi.query.did.dIDs.mockImplementation(async (address: string) => {
+      if (address === 'withDocumentStore') {
+        return mockChainQueryReturn('did', 'dIDs', [
+          key2,
+          key1,
+          '0x687474703a2f2f6d794449442e6b696c742e696f',
+        ])
+      }
+      return mockChainQueryReturn('did', 'dIDs', [key1, key2, null])
+    })
```

Comment:
this is identical to the one below, right? you could move it into a `beforeAll()` or just in the describe block.
if you change line 33 to `[...].mockReturnValueOnce(`, this would also make sure you don't completely overwrite the mock implementation.